sdk/include/reactos/ui/rosmsstyles.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * PROJECT:     ReactOS headers
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Support for PNG images in visual styles
+ * COPYRIGHT:   Copyright 2023 Ethan Rodensky <splitwirez@gmail.com>
+ */
+
+#if (DLL_EXPORT_VERSION >= _WIN32_WINNT_LONGHORN) && !defined(MSSTYLES_ACCEPT_PNG_IMAGES)
+/* If defined, uxtheme will use PNG images if present in a visual style,
+ * and built-in visual styles may include PNG images.
+ */
+#define MSSTYLES_ACCEPT_PNG_IMAGES
+#endif
```

Comment:
D:\a\reactos\reactos\src\sdk\include\reactos/ui/rosmsstyles.h(13) : fatal error RC1004: unexpected end of file found